src/app/shared/validators/time-format-validator.ts

```diff
@@ -0,0 +1,14 @@
+import { ValidatorFn, AbstractControl, ValidationErrors } from '@angular/forms';
+import { TIME_FORMAT_REGEX } from 'shared/constants/regex-constants';
+
+export function TimeFormatValidator(): ValidatorFn {
```

Comment:
Can we avoid an additional function wrapper for validator since we don't pass any parameters?